### PR TITLE
feat: release 3.6.1

### PR DESCRIPTION
**Pull request by Revolyssup**
### Description

Fixes # (issue)

### Checklist

- [ ] I have explained the need for this PR and the problem it solves
- [ ] I have explained the changes or the new features added to this PR
- [ ] I have added tests corresponding to this change
- [ ] I have updated the documentation to reflect this change
- [ ] I have verified that this change is backward compatible (If not, please discuss on the [APISIX mailing list](https://github.com/apache/apisix/tree/master#community) first)

### CODE REVIEW

**Comment by Revolyssup** [comment]
> #8999

@AlinsRan DONE

File: CHANGELOG.md [hunk 1]
```diff
@@ -73,6 +74,16 @@ title: Changelog
...
+## 3.6.1
+
+### Core
+
+- :sunrise: feat: upgrade lua-resty-healthcheck to 3.2.0：[#10307](https://github.com/apache/apisix/pull/10307)
+
+### Bugfixes
+
+- fix: keep healthcheck target state when upstream changes: [#10312](https://github.com/apache/apisix/pull/10312)
+
```
Comment (AlinsRan) [inline CHANGELOG.md:86]:
```suggestion
## 3.6.1

### Bugfixes

- fix: keep healthcheck target state when upstream changes: 
    - [#10307](https://github.com/apache/apisix/pull/10307)
    - [#10312](https://github.com/apache/apisix/pull/10312)

```

**Comment by Revolyssup** [comment]
@AlinsRan Pushed your change

File: docs/zh/latest/CHANGELOG.md [hunk 2]
```diff
@@ -73,6 +74,16 @@ title: CHANGELOG
...
 
+## 3.6.1
+
+### Core
+
+- :sunrise: 特征：升级 lua-resty-healthcheck 至 3.2.0: [#10307](https://github.com/apache/apisix/pull/10307)
+
+### Bugfix
+
+- 修复：当上游发生变化时保持健康检查目标状态： [#10312](https://github.com/apache/apisix/pull/10312)
```
Comment (AlinsRan) [inline docs/zh/latest/CHANGELOG.md:85]: This also needs to be changed.

**Review by AlinsRan — APPROVED** [review]
LGTM. Pls check ci.

File: docs/zh/latest/CHANGELOG.md [hunk 3]
```diff
@@ -73,6 +74,14 @@ title: CHANGELOG
 - [0.7.0](#070)
 - [0.6.0](#060)
 
+## 3.6.1
+
+### Bugfix
+
+- 修复：当上游发生变化时保持健康检查目标状态:
```
Comment (yzeng25) [inline docs/zh/latest/CHANGELOG.md:81]:
```suggestion
- 修复：当上游发生变化时保持健康检查目标状态：
```

Comment (yzeng25) [reply docs/zh/latest/CHANGELOG.md:81]: This should fix the lint check error

**Review by yzeng25 — APPROVED** [review]

**Comment by yzeng25** [comment]
CI passed, PTAL @Revolyssup @AlinsRan .

**Review by AlinsRan — APPROVED** [review]

**Review by soulbird — APPROVED** [review]

**Pull request merged**